xml/Microsoft.Vsa/VsaModule.xml

```diff
@@ -47,7 +47,7 @@
         <Parameter Name="bIsVsaModule" Type="System.Boolean" />
       </Parameters>
       <Docs>
-        <param name="bIsVsaModule">To be added.</param>
+        <param name="bIsVsaModule">True if the module is a VSA module, false otherwise</param>
```

Comment:
see how we do the formatting for other boolean params using the see langword tag

also check https://github.com/dotnet/dotnet-api-docs/wiki/Parameters%3A-Methods-and-constructors for language style rules